folly/synchronization/Rcu.h

```diff
@@ -383,8 +383,12 @@ class rcu_domain {
             syncTime, time, std::memory_order_relaxed)) {
```

Comment:
If the lock acquire fails, should we roll back `syncTime_` somehow?

Reply:
If the lock fails somebody else must be running synchronize, so the purpose of syncTime_ has been accomplished